### DISCUSSION TITLE
新版 Google Voice 將利用人工智慧自動標示、阻擋推銷內容等騷擾電話 目前僅支援美國用戶

### DISCUSSION THREAD

**Discussion by Ben6uo** [1101-GS3073-C/Python-Community-GS3073-C#550] [General]
常常接電話都是一推推銷的東西 真的很煩
https://www.cool3c.com/article/187379

**Comment by williamyen0623** [comment]
現在個資被賣來賣去，常常接到一堆莫名其妙的電話，不接又怕錯過什麼，接了又常常浪費時間，有了這個我相信能改善許多

**Comment by HsuChen88** [comment]
如果可以的話，能不能請Siri也幫我注意一下是不是推銷電話?

**Comment by boson13579** [comment]
身邊的親友和自己也常常借道一些車貸、銀行詐騙等等的騷擾電話，也因為這樣，養成只要是不明電話就不會接的習慣。只怕萬一是導師或同學有急事而漏接，會造成他們及自己的困擾。所以希望未來台灣也能自動標註是否是推銷電話。

**Comment by larry40305** [comment]
會打給我的人都是推銷電話，被阻擋以後就不會有人打給我了哈哈

**Comment by Bonus502530** [comment]
平時我不太會去理會陌生電話，我記得前一陣子參加系上活動我需要訂遊覽車，在訂完車後遊覽車公司說會再打電話來跟我做最後確認，不過當時並沒有明確說什麼時候會打給我，因此那個月只要有電話我都會接，那段時間接到的電話絕大多數都是推銷電話，如果台灣能有這項google服務應該能造福蠻多人的。
